lib/modules/platform/github/massage-markdown-links.spec.ts

```diff
@@ -87,4 +87,18 @@ describe('modules/platform/github/massage-markdown-links', () => {
       expect(massageMarkdownLinks(input)).toEqual(output);
     }
   );
+
+  it('replace url contains disappear text inline (colors)', () => {
+    const input = [
+      'pnpm rebuild accepts --store-dir by @&#8203;user in https://github.com/foo/bar/issues/1\n' +
+        'pnpm rebuild accepts --store-dir by @&#8203;UsEr in https://github.com/foo/bar/issues/2',
+    ].join('\n');
+    const res = massageMarkdownLinks(input);
+    expect(res).toEqual(
+      [
+        'pnpm rebuild accepts --store-dir by `@user` in [https://github.com/foo/bar/issues/1](https://togithub.com/foo/bar/issues/1)\n' +
```

Comment:
```suggestion
        'pnpm rebuild accepts --store-dir by `@user` in [foo/bar#1](https://togithub.com/foo/bar/issues/1)\n' +
```

we should do this in a follow up PR to reduce size some more

Reply:
@viceice  https://github.com/MaronHatoum/16921/pull/46